docs/getting-started/codacy-quickstart.md

```diff
@@ -4,9 +4,16 @@ description: Set up Codacy to automatically analyze your source code and identif
 
 # Codacy quickstart (5 min)
 
-Codacy automatically analyzes your source code and identifies issues as you go, helping you develop software more efficiently with fewer issues down the line. Through static code review analysis, Codacy notifies you of security issues, code coverage, code duplication, and code complexity in every commit and pull request.
+Codacy is an automated code quality and coverage platform that analyzes your source code and identifies issues as you go, helping your team ship robust software by scanning over 40 programming languages, such as JavaScript, Python, Java, C#, and PHP.
 
-Check out our product demo for an overview of the main Codacy features (recorded on February 4, 2022):
+By integrating with your Git provider, Codacy keeps track of your team’s work, analyzes relevant commits, highlights problems, suggests improvements, and protects your codebase from unwelcome changes. Codacy monitors the following metrics across your projects, from organization and repository overviews to individual files, pull requests, and commits:
+
+-   **Issues**: violations of a given rule, standard, convention, or best practice, from inconsistent code formatting to security risks
+-   **Complexity**: a measure of the number of execution paths through a program's source code
+-   **Duplication**: the amount of duplicated portions of code
+-   **Coverage**: the percentage of lines of code covered by automated tests
+
+Check out our product demo for an overview of Codacy's main features (recorded on February 4, 2022):
```

Comment:
Here, `Codacy's main features` sounds better than the previous variant, but consider it just a suggestion. It depends on whether we want to avoid the possessive `'s`

Reply:
This is looking very good! :star_struck: 

The only thing I noticed is that now, the sentence "To get started [...]" seems a bit hidden/lost under the video:

![image](https://user-images.githubusercontent.com/60105800/207588647-052d9999-2a06-424c-948e-8fa623e11b73.png)

What I'm thinking is that we could either:

* Pull that sentence from there and have it before the video, merging it with the "Check out our [...]". Something like this:

    > To get started, head to [codacy.com](https://www.codacy.com/) and click Get started. You can also check out our product demo for an overview of Codacy's main features (recorded on February 4, 2022):

* Expand a bit the sentence below the video so it properly introduces the steps in the sections. Like this:

    > To get started, head to [codacy.com](https://www.codacy.com/) and click Get started. Then, follow the instructions in the sections below.

    (I already feel like this alternative doesn't add much, though. :roll_eyes:)

* Move the video (together with the introducing sentence) between the current first and second paragraphs:

    > Coday is an automated [...]
    > 
    > Check out our product demo [...] + video
    > 
    > By integrating with your Git provider [...]

I think the last option is the one that could work better, but do try these out and pick the one that you prefer! :+1:

Reply:
OK. I applied the third suggestion and partially the second--it does flow a bit better with a transitional sentence.

Let me know if this works!

Reply:
Still, I wonder if placing the copy under the video may be the best option.

I'd personally scan the page copy before investing some minutes in a video presentation, but that's just my preference.

This would be an interesting candidate for an AB test...

Reply:
I'm going to approve the pull request now because from my point of view it "should work" like this. :sunglasses: 

But, as you say, this may boil down to the personal preferences of the users. We can wait for the opinion from @claudiacarpinteiro to help untie the decision. And, I propose that you spend some time figuring out if there's a way to perform A/B tests on the docs, as it's not the first time where we have had to make ambiguous decisions like this one. :ok_hand: I know there are some tools that the Design team uses, but I never investigated how they could apply to our docs site.

Reply:
I like the current option: the video keeps visible for those who prefer that media, and for those skipping it, they have the details right after.